ui/components/app/snaps/snap-ui-footer-button/snap-ui-footer-button.tsx

```diff
@@ -82,7 +83,7 @@ export const SnapUIFooterButton: FunctionComponent<
       {...props}
       size={ButtonSize.Lg}
       block
-      disabled={disabled}
+      disabled={requireScroll ? !buttonsEnabled : disabled}
```

Comment:
Shouldn't the `disabled` prop always take priority here?

```suggestion
      disabled={disabled || (requireScroll && !buttonsEnabled)}
```

Reply:
Yup, fixing.